api/app/views/notification_mailer/digest/daily/_annotation_event.html.erb

```diff
@@ -0,0 +1,6 @@
+<% annotation = event.subject %>
+<% url = "#{@client_url}/read/#{annotation.text.slug}/section/#{annotation.text_section_id}#annotation-#{annotation.id}" %>
```

Comment:
We gotta get the URL building out of these views. If URLs change in the client, it's going to be more difficult than it should be to track these down. Build a service, or somehow centralize the URL generation instead of embedding this in the view layer.